src/DynamoCore/Graph/Workspaces/SerializationConverters.cs

```diff
@@ -107,11 +108,16 @@ public override object ReadJson(JsonReader reader, Type objectType, object exist
                type = Type.GetType(obj["$type"].Value<string>());
                typeName = obj["$type"].Value<string>().Split(',').FirstOrDefault();
 
-               if (typeName.Contains("ZeroTouch"))
-               {
-                    // This assemblyName does not usually contain version information...
+                if (typeName.Contains("ZeroTouch"))
```

Comment:
can we be specific about this-  it's possible anyone could make a nodeModel and name it `ZeroTouch` lets use our entire namespace and type name - including the assembly name would be fine as well for precision.